azure-communication-ui/azure-communication-ui/src/test/java/com/azure/android/communication/ui/redux/reducer/LocalParticipantReduxStateReducerUnitTest.kt

```diff
@@ -219,7 +219,7 @@ internal class LocalParticipantReduxStateReducerUnitTest {
             null
         )
         val error = CallCompositeError(
-            CommunicationUIErrorEvent.TURN_CAMERA_OFF,
+            CommunicationUIErrorCode.TURN_CAMERA_OFF,
             Throwable("CameraOff failed")
```

Comment:
Should we keep the error messages consistent overall? -- CameraOff has failed instead of CameraOff failed

Reply:
It is just a unit test, but sure.